### PR TITLE
Release notes for 2021.1

### PR DESCRIPTION
**Pull request by lbalicki**
I started working on the 2021.1 release notes file. Feel free to add to the file or change my suggested PRs mentioned in the notes.

### CODE REVIEW

File: docs/source/release_notes/2021.1.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+`[#1149] <https://github.com/pymor/pymor/pull/1149>`_: Riccati equation solvers
+for small and dense matrices, Bernoulli matrix equation solver, new options for
+for pyMORs eigensolver such as shift-and-invert mode and :math:`\mathcal{L}_2`-norm
+computation for |LTIModels|. The new methods and reductors are showcased in
+:doc:`../tutorial_unstable_lti_systems`.
+
+Modal truncation
+~~~~~~~~~~~~~~~~
+
+Output reductor using ANNs
```
Comment (HenKlei) [inline docs/source/release_notes/2021.1.rst:41]: I would not call that a "highlight" of the release. For me it is more like an additional new feature. If #1383 makes it into the release, that would be much more interesting I think.

File: docs/source/release_notes/2021.1.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+Additional new features
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Improvements to the HAPOD algorithm
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Empirical Interpolation improvments
```
Comment (HenKlei) [inline docs/source/release_notes/2021.1.rst:54]:
```suggestion
Empirical Interpolation improvements
```

**Review by HenKlei — CHANGES_REQUESTED** [review]

File: docs/source/release_notes/2021.1.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+computation for |LTIModels|. The new methods and reductors are showcased in
+:doc:`../tutorial_unstable_lti_systems`.
+
+Modal truncation
+~~~~~~~~~~~~~~~~
+
+Output reductor using ANNs
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Time dependent parameter values
```
Comment (pmli) [inline docs/source/release_notes/2021.1.rst:44]:
```suggestion
Time-dependent parameter values
```

File: docs/source/release_notes/2021.1.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Methods for unstable LTI systems
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Many popular system-theoretic model order reduction methods are not applicable
+to unstable LTI systems out of the box. In
+`[#1149] <https://github.com/pymor/pymor/pull/1149>`_ two reductors and several
+methods for working with and analyzing unstable |LTIModels| have been added.
+The :class:`~pymor.reductors.bt.FDBTReductor` allows for applying the balanced
+truncation technique to unstable systems by performing a Bernoulli stabiliztion
+before the using the classical BT method. The :class:`~pymor.reductors.h2.GapIRKAReductor`
+aims to compute a reduced order model such that the approximation error with
```
Comment (pmli) [inline docs/source/release_notes/2021.1.rst:29]:
```suggestion
aims to compute a reduced-order model such that the approximation error with
```

File: docs/source/release_notes/2021.1.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
+
+pyMOR 2021.1 (September 30, 2021)
+--------------------------------
```
Comment (pmli) [inline docs/source/release_notes/2021.1.rst:3]:
```suggestion
pyMOR 2021.1 (September 30, 2021)
---------------------------------
```

**Comment by pmli** [comment]
Ok, great. I can add some text for MT and LTI matrices export.

File: docs/source/release_notes/2021.1.rst (hunk 1 above)
Comment (lbalicki) [reply docs/source/release_notes/2021.1.rst:41]: Agreed. We should change the release notes summary as soon as #1383 is merged.

Comment (renefritze) [reply docs/source/release_notes/2021.1.rst]: #1383 didn't make it, so no change here.

**Pull request merged**